src/components/BookList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import ContentBook from './ContentBook';
+import BookForm from './BookForm';
+
+const BookList = () => {
+  const books = useSelector((state) => state.books);
+
+  return (
+    <div>
+      <ul>
+        {books.map((book) => (
+          <ContentBook
+            book={book}
+            key={book.item_id}
+          />
+        ))}
+      </ul>
+      <BookForm />
+    </div>
+  );
+};
+
```

Comment:
- [ ] Kinldy as was requested by the previous reviewer. Use `useEffect` to render the list of books 

`useEffect(() => {
    your dispatch function
  }, [])`

Reply:
included useEffect